src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0" ?>
 
 <container xmlns="http://symfony.com/schema/dic/services"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
All these xml changes are due to a bad merge when updating from upstream, I'll make sure to get the correct xml files and then adding the service config for the L18n validator